xstream/src/test/com/thoughtworks/xstream/io/naming/StaticNameCoderTest.java

```diff
@@ -0,0 +1,48 @@
+package com.thoughtworks.xstream.io.naming;
+
+import com.thoughtworks.xstream.io.naming.StaticNameCoder;
+import junit.framework.TestCase;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class StaticNameCoderTest extends TestCase {
+
+  public void testDecodeAttribute() {
+    final HashMap<String, String> node = new HashMap<>();
+    final HashMap<String, String> attribute = new HashMap<>();
+    StaticNameCoder staticNameCoder = new StaticNameCoder(node, attribute);
+    assertEquals("A", staticNameCoder.decodeAttribute("A"));
```

Comment:
add blank lines around this assert to separate it

Reply:
change the string `A` to `non-existing attribute`

Reply:
apply similarly to other tests

Reply:
Done